extensions/amp-carousel/0.1/carousel.js

```diff
@@ -155,6 +155,7 @@ export class AmpCarousel extends BaseCarousel {
   doLayout_(pos) {
     this.withinWindow_(pos, cell => {
       this.scheduleLayout(cell);
+      this.scheduleResume(cell);
```

Comment:
Should probably wait for the element to be laid out before letting either `#pause` or `#resume` do anything.